packages/node_modules/@node-red/editor-client/src/js/ui/userSettings.js

```diff
@@ -101,10 +101,25 @@ RED.userSettings = (function() {
         RED.tray.show(trayOptions);
     }
 
+    function locale2Name(lc) {
+        var name = RED._("languages."+lc);
+        return {text: (name ? name : lc), val: lc};
+    }
+
+    function compText(a, b) {
```

Comment:
Can this just use `localeCompare`? We use that elsewhere for this sort of thing without issue.
```
function compareText(a, b) {
   return a.text.localeCompare(b.text);
}
```

Reply:
changed to use `localeCompare`. Thank you!